### PR TITLE
Update about-github-pages.md

### PR DESCRIPTION
**Pull request by jw-12138**
### Why:

Lately i'm working on an unofficial Chinese translation of GitHub Pages docs due to the incompletion of the original and currently there is no way to contribute translations to the docs. So, I hope that's ok to you guys.

### What's being changed:

The thing is, I'm seeing that this document is using both `<user>` and `<username>` as subdomain pronouns among the context. It's not a big problem, but in my opinion, this might cause some unnecessary confusion to some users.   

If it was intended to be so, I'd like to know why?  
Thank you!

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/working-with-github-pages/about-github-pages.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ Organization owners can disable the publication of {% data variables.product.pro
...
+To publish a user site, you must create a repository owned by your user account that's named {% if currentVersion == "free-pro-team@latest" %}`<username>.github.io`{% else %}`<username>.<hostname>`{% endif %}. To publish an organization site, you must create a repository owned by an organization that's named {% if currentVersion == "free-pro-team@latest" %}`<organization>.github.io`{% else %}`<organization>.<hostname>`{% endif %}. {% if currentVersion == "free-pro-team@latest" %}Unless you're using a custom domain, user and organization sites are available at `http(s)://<username>.github.io` or `http(s)://<organization>.github.io`.{% endif %}
```
Comment (ST15392) [inline content/github/working-with-github-pages/about-github-pages.md:36]: Great change!

**Review by ghost — APPROVED** [review]
I agree that <user> should be replaced with <username>, hence approval.

**Comment by janiceilene** [comment]
👋 Thanks for opening a PR @jw-12138! This is now triaged for review. Sorry about the delay, I somehow missed this PR 💛

**Review by dennispan — APPROVED** [review]
Looks good 👍

**Comment by hubwriter** [comment]
@jw-12138 - Thank you for this pull request. That's a good change. Currently were not merging pull requests, but we'll start doing so again very soon and I'll get this merged. Thanks again for helping to improve GitHub's documentation. 👍 

You're right about the non-English versions of pages. Unfortunately, we're not currently able to accept pull requests for translated content. Our translation process involves an integration with an external service at crowdin.com, where all translation activity happens. We hope to eventually open up the translation process to the open-source community, but we're not there yet.

**Review by hubwriter — APPROVED** [review]

**Comment by jw-12138** [comment]
> @jw-12138 - Thank you for this pull request. That's a good change. Currently were not merging pull requests, but we'll start doing so again very soon and I'll get this merged. Thanks again for helping to improve GitHub's documentation. 👍
> 
> You're right about the non-English versions of pages. Unfortunately, we're not currently able to accept pull requests for translated content. Our translation process involves an integration with an external service at crowdin.com, where all translation activity happens. We hope to eventually open up the translation process to the open-source community, but we're not there yet.

Thanks for replying!  
This is in fact the first official PR I did, looks like it's a good start!  
I'll keep doing my best as you guys doing out there! 👍  
  
Also, about `Crowdin`, It might be offensive to say so, a good translation should use both experiences and skills, I don't think every `Crowdin` user is actually **using** the product they translate...do they? (🤣 that's a lot on me). But hey, it's a very cool platform! I've registered my account and hoping to get started to do something very soon.

Anyway, Thank you!

**Pull request merged**